src/web/routes.rs

```diff
@@ -93,6 +93,7 @@ pub(super) fn build_routes() -> Routes {
         "/crate/:name/:version/target-redirect/*",
         super::rustdoc::target_redirect_handler,
     );
+    routes.internal_page("/api/v1/badges", super::api::badge_handler_v1);
```

Comment:
We shouldn't add more reserved prefixes.

I propose all new top-level routes should be prefixed by `-`, for example:

```
/-/api/v1/badges
```

Reply:
What does the `-` do, I'm not familiar with it

Reply:
It's just a name that can't be registered on crates.io.